python/pyspark/pandas/groupby.py

```diff
@@ -44,6 +43,7 @@
 )
 import warnings
 
+import numpy as np
```

Comment:
`numpy` in the docstring was imported in `def _test()`, so no need to import it here